source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -728,3 +728,23 @@ def _filter_too_much_connections(journeys, instance, request):
             if get_nb_connections(j) > max_connections_allowed:
                 logger.debug("the journey {} has a too much connections, we delete it".format(j.internal_id))
                 mark_as_dead(j, is_debug, "too_much_connections")
+
+
+def remove_excess_tickets_after_culling(response):
```

Comment:
minor:
```suggestion
def remove_excess_tickets(response):
```
As culling is not especially linked to the function itself (clean comments and log as well if you do it).